src/datasources/config-api/config-api.service.ts

```diff
@@ -54,11 +58,8 @@ export class ConfigApi implements IConfigApi {
     url?: string,
   ): Promise<SafeApp[]> {
     try {
-      const cacheKey = `safe_apps`;
-      const field = `${chainId}_${clientUrl}_${url}`;
       return await this.dataSource.get(
-        cacheKey,
-        field,
+        CacheRouter.getSafeAppsCacheDir(chainId, clientUrl, url),
```

Comment:
nitpick: see my comment regarding extracting the `cacheDir` parameter.

Additionally, to extract the URL and params here.

Reply:
Changed in [76724b7](https://github.com/5afe/safe-client-gateway-nest/pull/285/commits/76724b74b6eeed8fb5ab522bf72045f6ffe0f730)